OSDOCS-14695-1:Update the z-stream RNs for 4.15.51 (#93553)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.15.51
[id="ocp-4-15-51_{context}"]
=== RHSA-2025:7698 - {product-title} 4.15.51 bug fix and security update

Issued: 21 May 2025

> {product-title} release 4.15.51, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7698[RHSA-2025:7698] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/2025:7700[RHBA-2025:7700] advisory.

Reviewer: Both advisory links give a 404 error. Please reconfirm the links once the links are published.

Reviewer 2: The links seem to be working now ...

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.15.51 --pullspecs
----

[id="ocp-4-15-51-known-issues_{context}"]
==== Known issues

* The grandmaster clock (T-GM) transitions to the Locked state too soon. This is a known issue that happens before the Digital Phase-Locked Loop (DPLL) completes its change to the Locked-HO-Acquired state, and after the Global Navigation Satellite Systems (GNSS) time source is restored. (link:https://issues.redhat.com/browse/OCPBUGS-55466[*OCPBUGS-55466*]). 

[id="ocp-4-15-51-bug-fixes_{context}"]
==== Bug fixes

> * Previously, when viewing the list of installed Operators, an Operator appeared twice in the list. This duplication occurred when the currently selected project matched an Operator's default namespace while copied cluster service versions (CSVs) were disabled in the Operator Lifecycle Manager (OLM). With this release, the Operator appears once. (link:https://issues.redhat.com/browse/OCPBUGS-55733[OCPBUGS-55733]). 

Reviewer 2: ```suggestion
* Previously, when you viewed the list of installed Operators, an Operator appeared twice in the list because the currently selected project matched an Operator's default namespace while the copied cluster service versions (CSVs) were disabled in the Operator Lifecycle Manager (OLM). With this release, the Operator appears once. (link:https://issues.redhat.com/browse/OCPBUGS-55733[OCPBUGS-55733]). 
```

Reviewer 2: The fix seems to be missing?

https://redhat-documentation.github.io/supplementary-style-guide/#_bug_fix

> * Previously, when a validating webhook was created for a user or group on a resource managed by the OpenShift OAuth API server, the webhook was not triggered because the OAuth APIs were not reachable through the konnectivity tunnel. With this release, the issue is resolved by adding a `konnectivity-proxy` sidecar to the OAuth API server, enabling proper communication between the server and the data plane. (link:https://issues.redhat.com/browse/OCPBUGS-55266[OCPBUGS-55266]).

Reviewer 2: ```suggestion
* Previously, when you created a validating webhook for a user or group on a resource managed by the OpenShift OAuth API server, the webhook was not triggered because the OAuth APIs were not reachable through the konnectivity tunnel. With this release, adding a `konnectivity-proxy` sidecar to the OAuth API server resolves the issue and enables proper communication between the server and the data plane. (link:https://issues.redhat.com/browse/OCPBUGS-55266[OCPBUGS-55266]).
```
